File: src/main/java/com/ksptool/ql/biz/model/vo/RecoverRpChatHistoryVo.java
package com.ksptool.ql.biz.model.vo;

import lombok.Data;
import java.util.Date;

@Data
public class RecoverRpChatHistoryVo {

    //对话历史ID
    private Long id;

    // 角色名称
    private String name;

    // 头像路径
    private String avatarPath;

    //消息类型：0-用户消息，1-AI消息
    private Integer type;

    //消息内容-原始消息(展示给用户看的消息,不含系统Prompt等)
    private String rawContent;
    
    //消息创建时间
    private Date createTime;
}

File: src/main/resources/views/aether/src/entity/RecoverRpChatHistoryVo.ts
/**
 * 恢复角色扮演聊天历史记录项 VO
 */
export default interface RecoverRpChatHistoryVo {

    /**
     * 对话历史ID
     */
    id: string; // Corresponds to Java Long

    /**
     * 角色名称
     */
    name: string;

    /**
     * 头像路径
     */
    avatarPath: string;


    //消息类型：0-用户消息，1-AI消息
    type: 0 | 1;

    /**
     * 消息内容
     * Corresponds to Java String rawContent
     */
    rawContent: string;

    /**
     * 消息创建时间 (ISO 8601 格式字符串)
     * Corresponds to Java Date
     */
    createTime: string; 

} 